Formula/mypy.rb

```diff
@@ -2,8 +2,8 @@ class Mypy < Formula
   desc "Experimental optional static type checker for Python"
   homepage "http://www.mypy-lang.org/"
   url "https://github.com/JukkaL/mypy.git",
-    :tag => "v0.4.2",
-    :revision => "b8a2df91707bc3d5c9c3677c8017b614e41d017a"
+      :tag => "v0.4.3",
```

Comment:
I'm happy either way but this seems to be @DomT4's preferred indentation for these based on inspection of other formulae. (Please correct me if I'm wrong @DomT4!)

We also have this exact same discrepancy with `go_resource` blocks right now where some of them are like this
(a)
```
  go_resource "github.com/BurntSushi/toml" do
    url "https://github.com/BurntSushi/toml.git",
    :revision => "f0aeabca5a127c4078abb8c8d64298b147264b55"
  end
```
(b)

Some are like this
```
  go_resource "github.com/BurntSushi/toml" do
    url "https://github.com/BurntSushi/toml.git",
      :revision => "f0aeabca5a127c4078abb8c8d64298b147264b55"
  end
```
and some are like this
(c)
```
  go_resource "github.com/BurntSushi/toml" do
    url "https://github.com/BurntSushi/toml.git",
        :revision => "f0aeabca5a127c4078abb8c8d64298b147264b55"
  end
```
With tags there's also a fourth option, where people stick both on the same line, again with the indentation starting at random locations.

From what I can gather (a) is preferred by no one but is being introduced by the gdm tool, which is doing them wrong, but that there is a pervasive lack of consistency between (b) and (c), including with tags, as above.

It would be nice IMO to have a rule or rules for this that we follow consistently, whatever those rule(s) might be :)